### PR TITLE
All Panjab typos changed to Punjab

### LINKED ISSUE #50: Change Panjab to Punjab
Change type at all locations in the repo from Panjab to Punjab

### PR DESCRIPTION
**Pull request by habeeba-naaz**
Please let me know if any corrections to be made..
It fixes #50.

### CODE REVIEW

File: bull/pdl-queue/consumer.js [hunk 1]
```diff
@@ -1,175 +1,221 @@
...
+          var data = new Buffer(body.data);
+          img.file(filename, data.toString("base64"), { base64: true });
+        }
+      })
+      .catch(function(err) {
+        --no_of_pages;
+      });
+  };
+  for (let i = 1; i <= temp_pages; ++i) {
+    const str = `http://www.punjabdigilib.org/images?ID=${id}&page=${i}&pagetype=null&Searched=W3GX`;
```
Comment (rkpattnaik780) [inline bull/pdl-queue/consumer.js]: The URLs are not supposed to be changed. The website itself is : http://www.panjabdigilib.org/webuser/searches/mainpage.jsp 
Changing the URL will give us a 404 Error.

File: bull/pdl-queue/consumer.js [hunk 2]
```diff
@@ -1,175 +1,221 @@
...
+  ] = `LOW ${process.env.access_key}:${process.env.secret_key}`;
+  headers["Content-type"] = "application/zip";
+  headers["Content-length"] = byteLength;
+  headers["X-Amz-Auto-Make-Bucket"] = 1;
+  headers["X-Archive-meta-collection"] = "opensource";
+  headers["X-Archive-Ignore-Preexisting-Bucket"] = 1;
+  headers["X-archive-meta-identifier"] = title;
+  headers["X-archive-meta-mediatype"] = "texts";
+  headers["X-archive-meta-uploader"] = "bub.wikimedia@gmail.com"; //To be added
+  headers["X-archive-meta-contributor"] = "Punjab Digital Library"; //To be added
```
Comment (rkpattnaik780) [inline bull/pdl-queue/consumer.js]: It should be `Panjab` here as well. The changes should be made only where 'Panjab' is getting displayed, not while interacting with the APIs.

File: README.md [hunk 3]
```diff
@@ -1,29 +1,34 @@
 ## BUB2
-A book uploader bot that transfers documents from public libraries such as Google Books, and Panjab Digital Library etc to Internet Archive. Built for Wikimedia Tool Labs. Check out [BUB2 on Toolforge](https://bub2.toolforge.org)!
+
+A book uploader bot that transfers documents from public libraries such as Google Books, and Punjab Digital Library etc to Internet Archive. Built for Wikimedia Tool Labs. Check out [BUB2 on Toolforge](https://bub2.toolforge.org)!
```
Comment (rkpattnaik780) [inline README.md]: Make it `Punjab (Panjab)`.

Comment (rkpattnaik780) [reply README.md]: This suggestion has not been addressed.

File: components/Books.js [hunk 4]
```diff
@@ -41,7 +41,8 @@ export default class Books extends React.Component {
         break;
 
       case "pn":
-        url = "http://www.panjabdigilib.org/webuser/searches/displayPageContent.jsp?ID=xxxx&page=x&CategoryID=x&Searched=xxxx";
+        url =
+          "http://www.punjabdigilib.org/webuser/searches/displayPageContent.jsp?ID=xxxx&page=x&CategoryID=x&Searched=xxxx";
```
Comment (rkpattnaik780) [inline components/Books.js]: Change the URL here.

File: components/Books.js [hunk 5]
```diff
@@ -215,31 +214,32 @@ export default class Books extends React.Component {
...
             <input
               id="bookid"
               name="bookid"
-              type={this.state.option === 'gb' ? 'text' : 'url'}
-              placeholder={this.state.option === 'gb' ? "At46AQAAMAAJ" : 'http://www.panjabdigilib.org/webuser/searches/displayPageContent.jsp?ID=2833&page=1&CategoryID=3&Searched=W3GX'}
+              type={this.state.option === "gb" ? "text" : "url"}
+              placeholder={
+                this.state.option === "gb"
+                  ? "At46AQAAMAAJ"
+                  : "http://www.punjabdigilib.org/webuser/searches/displayPageContent.jsp?ID=2833&page=1&CategoryID=3&Searched=W3GX"
```
Comment (rkpattnaik780) [inline components/Books.js]: Change URL here.

File: README.md (hunk 3 above)
Comment (habeeba-naaz) [reply README.md]: In README.md...i already changed 'panjab' to 'punjab'.In original file only one paragraph is present to be changed.(making 'panjab' to 'punjab').But here it's showing same repeated paragraph without changes(panjab remains as it is).

**Comment by rkpattnaik780** [comment]
The URLs have been changed but as discussed in https://github.com/coderwassananmol/BUB2/pull/65#pullrequestreview-370748785 , you are not supposed to lint here. I suggest you to revert back the linting changes. And please update the PR by tonight, it is already taking too long.

**Comment by habeeba-naaz** [comment]
I am not getting what is linting changes...?Are you Asking about links changes to keep it as panjab? If so I already changed links to panjab.

**Comment by rkpattnaik780** [comment]
> I am not getting what is linting changes...?Are you Asking about links changes to keep it as panjab? If so I already changed links to panjab.

Let me be more clear, from linting changes I mean the changes in indentation, ensuring double quotes and proper spacing. I am asking you to revert those changes. It is good to incorporate these changes but we could make another issue for it and assign it to you.

**Comment by habeeba-naaz** [comment]
I got it...the thing is my vscode, I included that plugin to automatically include formatting...so it did..I just took care of typos which you specified...so It’s difficult for me to keep those things back...

**Comment by rkpattnaik780** [comment]
> I got it...the thing is my vscode, I included that plugin to automatically include formatting...so it did..I just took care of typos which you specified...so It’s difficult for me to keep those things back...

Okay, be careful from next time onwards, I will approve it. The only thing you need to do is solve the merge conflicts that have arisen.

**Comment by habeeba-naaz** [comment]
Please let me know why README.md is showing conflict...how to do?

**Comment by habeeba-naaz** [comment]
Thank you for approving...why README.md file conflict came?

**Comment by rkpattnaik780** [comment]
> Thank you for approving...why README.md file conflict came?

Feel free to ping me on slack. I will guide you through.

**Comment by habeeba-naaz** [comment]
Sure

**Comment by habeeba-naaz** [comment]
Slack I’d please

**Comment by rkpattnaik780** [comment]
> Slack I’d please

Just search `Ramakrishna Pattnaik` in the DMs.

**Pull request merged**